titan-compiler/lexer.lua

```diff
@@ -95,6 +95,14 @@ do
                 return open == close
             end)
 
+    local decimal_escape = P("1") * R("09") * R("09") +
+        P("2") * R("04") * R("09") * -R("09") +
+        P("2") * P("5") * R("05") * -R("09") +
+        P("0") * R("09") * R("09") * -R("09") +
+        R("09") * R("09") * -R("09") +
+        R("09") * -R("09") +
+        R("09") * T(labels.MalformedEscape_decimal)
+
```

Comment:
Acho que tem que tirar aqueles `-R("09")`. Da forma como está ele rejeita coisas como `\"2490`, que deveria aceitar.

Reply:
I guess this is in the eye of the beholder, I am not fond of overusing math-time captures myself... will fix the second point.